app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -48,8 +52,25 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
             />
+            <div className="w-full flex-col justify-between">
+              <Select
+                required
+                label="Assigned Contact"
+                name="contactList"
```

Comment:
Suffixing with the `List` keyword is unnecessary. Rather name this as `contacts`. But here, you are only selecting one contact. Thus having the `List` was wrong in the first place. Here, singular, `contact` should be the name.